infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/event/ContainerEvent.java

```diff
@@ -62,6 +70,7 @@ public boolean equals(Object o) {
     ContainerEvent that = (ContainerEvent) o;
     return Objects.equals(podName, that.podName)
         && Objects.equals(containerName, that.containerName)
+        && Objects.equals(reason, that.reason)
```

Comment:
Can you also add the reason to the hashcode method? It is common to have them aligned.

Reply:
done, thanks for the pointer